routes/vectordata.py

```diff
@@ -18,6 +20,37 @@
 
 data_api = Blueprint("data_api", __name__)
 
+# Precompute all of the communities filtered by extent
+geojson_names = ["alaska", "blockyAlaska", "mizukami", "slie"]
+extent_filtered_communities = {}
+
+all_communities_full = asyncio.run(
```

Comment:
This one duplicates the functionality of the other endpoint that does this, suggest calling the function instead of doing a data fetch